libshaderc_util/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ target_include_directories(shaderc_util
   PUBLIC include PRIVATE ${glslang_SOURCE_DIR})
 find_package(Threads)
 target_link_libraries(shaderc_util PRIVATE
```

Comment:
Do we remember why we declared this as a PRIVATE library dependency. I think it is fairly clearly a PUBLIC one as anyone who links shader_util must also link to glslang etc etc etc.

IIRC If we make this public we can remove all of the glslang-related includes from glslc.